packages/amplify-graphql-model-transformer/rds-patching-lambda/src/index.ts

```diff
@@ -0,0 +1,61 @@
+import { LambdaClient, UpdateFunctionConfigurationCommand, UpdateFunctionConfigurationCommandOutput } from "@aws-sdk/client-lambda";
+
+const snsEventSource = 'aws:sns';
+
+type LayerConfig = {
+  layerArn: string;
+  region: string;
+};
```

Comment:
At least this type is shared across your two lambdas, if there's sizable shared resources/shapes/utils I'd consider moving these to a single bundle w/ multiple entry points.

Reply:
Though these types are same, right now they live in two different lambdas. We don't share any details between these lambdas. Probably we can export all these types to a common package but let's keep it outside the scope of this PR.